### PR TITLE
Eprod 619 update tokenlist

### PR DESCRIPTION
**Pull request by Leleking**

### CODE REVIEW

File: src/tokenlist.json [hunk 1]
```diff
@@ -91,23 +81,104 @@
...
+      "symbol": "EMC",
+      "fee": 100000
+    },
+    {
+      "id": "wrett-waaaa-aaaaq-aabda-cai",
+      "index_canister": "x3lrj-uqaaa-aaaaq-aabea-cai",
+      "logo": "https://raw.githubusercontent.com/infinity-swap/token-lists/main/logos/mod.png",
+      "decimals": 8,
+      "standard": "ICRC1",
+      "name": "Modclub",
```
Comment (blutooth) [inline src/tokenlist.json:109]: why is modclub repeated twice?

File: src/tokenlist.json [hunk 2]
```diff
@@ -91,23 +81,104 @@
...
-      "name": "NOBLEBLOCKS",
-      "symbol": "NOBL",
-      "fee": 10
+      "decimals": 8,
+      "name": "Nuance",
+      "symbol": "NUA",
+      "fee": 100000
     },
     {
-      "id": "jwcfb-hyaaa-aaaaj-aac4q-cai",
```
Comment (blutooth) [inline src/tokenlist.json:104]: why is origyn deleted?

File: src/tokenlist.json (hunk 1 above)
Comment (Leleking) [reply src/tokenlist.json:109]: taking a look into this

Comment (Leleking) [reply src/tokenlist.json]: I have removed the dupliccates

File: src/tokenlist.json (hunk 2 above)
Comment (Leleking) [reply src/tokenlist.json:104]: I have restore all tokens

**Review by blutooth — APPROVED** [review]

**Review by EstherOA — APPROVED** [review]

**Pull request merged**